### PR TITLE
Revert #1815

### PR DESCRIPTION
**Pull request by jlowin**
#1815 introduced bugs related to passing non-slugified variables an environment variables, this PR reverts it (automatic reversion is unavailable due to subsequent changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,9 +52,7 @@ Released on Dec 11, 2019.
 
 ### Enhancements
 
-- Add convenience `parents()` and `children()` classmethods to all State objects for navigating the hierarchy - [#1784](https://github.com/PrefectHQ/prefect/pull/1784)
```
Comment (cicdw) [inline CHANGELOG.md]: Looks like you're removing a valid entry here

Comment (jlowin) [reply CHANGELOG.md]: I think it was actually doubled in the original PR then - it's here a few lines down https://github.com/PrefectHQ/prefect/pull/1854#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL59

Comment (cicdw) [reply CHANGELOG.md]: Ah!

**Comment by jlowin** [comment]
@cicdw is the requested change a different one from the one we discussed?

**Comment by cicdw** [comment]
Yup - approving and rerunning appveyor right now...

**Pull request merged**